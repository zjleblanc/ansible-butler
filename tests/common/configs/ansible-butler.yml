---
execution_environment:
  init:
    ee_base_image: registry.redhat.io/ansible-automation-platform-22/ee-supported-rhel8:latest
    ee_builder_image: registry.redhat.io/ansible-automation-platform-22/ee-supported-rhel8:latest

directory:
  init:
    folders:
      - name: inventories
        folders:
          - name: example-inventory
            files:
              - hosts
            folders:
              - name: group_vars
              - name: host_vars
      - name: library
      - name: module_utils
      - name: filter_plugins
      - name: test_plugins
      - name: roles
    files:
      - playbook.yml.j2
      - ansible.cfg

role: {}

playbook:
  update:
    modules:
      smart_device:
        redirect: zjleblanc.kasa.smart_device
