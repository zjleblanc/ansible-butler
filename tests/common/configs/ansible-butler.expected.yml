execution_environment:
  init:
    version: 2
    ansible_config: ansible.cfg
    ee_base_image: registry.redhat.io/ansible-automation-platform-22/ee-supported-rhel8:latest
    ee_builder_image: registry.redhat.io/ansible-automation-platform-22/ee-supported-rhel8:latest
    prepend_build_steps:
      - RUN pip3 install --upgrade pip setuptools
    append_build_steps:
      - RUN echo 'Build completed (ansible-butler)'

directory:
  init:
    files:
    - playbook.yml.j2
    - ansible.cfg
    folders:
    - folders:
      - files:
        - hosts
        folders:
        - name: group_vars
        - name: host_vars
        name: example-inventory
      name: inventories
    - name: library
    - name: module_utils
    - name: filter_plugins
    - name: test_plugins
    - name: roles

playbook:
  update:
    modules:
      smart_device:
        redirect: zjleblanc.kasa.smart_device
        
role: {}
